nodes/exchange/bezier_in.py

```diff
@@ -14,31 +14,32 @@
 from sverchok.data_structure import updateNode, zip_long_repeat, split_by_count
 from sverchok.utils.curve.algorithms import concatenate_curves
 from sverchok.utils.curve.bezier import SvCubicBezierCurve
+from sverchok.utils.nodes_mixins.show_3d_properties import Show3DProperties
 
 class SvBezierInCallbackOp(bpy.types.Operator):
 
     bl_idname = "node.sv_bezier_in_callback"
     bl_label = "Bezier In Callback"
     bl_options = {'INTERNAL'}
 
-    node_name: StringProperty(default='')
-    tree_name: StringProperty(default='')
+    idname: StringProperty(default='')
```

Comment:
@portnov i complete want to convert all `idname` and `idtree` to `node_name` and `tree_name`.  after this :)

Reply:
Then why are you renaming tree_name to idtree here?

Reply:
because i want to focus on this stupid operator first, there are many references to idname and idtree.